main/config.c

```diff
@@ -205,6 +205,37 @@ uint32_t config_ble_passkey_get(const char *mac)
     return 0;
 }
 
+/* Ethernet Configuration */
+uint8_t config_eth_enabled_get(void)
```

Comment:
With the comment above, this function will change to check if the `eth` field exists or `wifi`.
Maybe change it to `config_network_type_get()` which will return an enum value of `NETWORK_TYPE_WIFI` or `NETWORK_TYPE_ETH`.